src/app/admin/admin-campaign-detail/admin-campaign-detail.component.html

```diff
@@ -97,7 +97,7 @@ <h3 class="text-2xl text-indigo-900">
                 <span>Asset: <b>{{ campaignData.asset.name }}</b></span>
                 <span>Price per token: <b>{{ campaignData.stats.tokenPrice | currencyDefault }}</b></span>
                 <span>Tokens sold: <b>{{ campaignData.stats.tokensSold | number }}</b></span>
-                <span>Token balance: <b>{{ campaignData.stats.tokenBalance | number }}</b></span>
+                <span>Token balance: <b>{{ campaignData.stats.tokensAvailable | number }}</b></span>
```

Comment:
Note: `tokenBalance` is the total number of tokens that were added to campaign (sold + unsold) while `tokensAvailable` is the current amount of unsold tokens. Since we are displaying the number of sold tokens above this number, it makes sense to me to display the number of remaining tokens. @mightymatth Do you think we should to it like this and possibly rename "Token balance" to something like "Unsold tokens" or is "Token balnce" ok?

Reply:
I'm discussing this with @filipdujmusic atm.

Reply:
We hopefully fixed these calculations in stats in [this](https://github.com/AMPnet/amptzr/commit/233c585ddf88fccb3626a81e9e101fb748cbfb5e) commit. Merge the latest master.